examples/style_sheets/plot_fivethirtyeight.py

```diff
@@ -12,13 +16,16 @@
 # Fixing random state for reproducibility
 np.random.seed(19680801)
 
+fig, ax = plt.subplots()
```

Comment:
You might need to do this within the context to ensure that general figure styling matches the style.